hicommonwealth/commonwealth
# 7136 infiniteloopbug

## Link to Issue
Closes: #7136 

## Description of Changes
- Got rid of the usage of `useBrowserWindow`
- Using our breakpoint helper instead

## "How We Fixed It"

- Because `useBrowserWindow` is dependent on the window size, it was firing the query on every resize therefore re-rendering the component and thus firing the API call
- By switching to the helper `isWindowSmallInclusive` we solve the rapid firing of the endpoint 

## Test Plan
- Go to :community/discussions
- Open dev tools network tab
- Ensure the `thread.getBulkThreads` is not firing more than needed


## Other Considerations

- [video](https://github.com/hicommonwealth/commonwealth/assets/126922418/01e4c52b-9e91-4194-9ad7-bb418449023f)
